telco-hub/install/agent-config.yaml

```diff
@@ -0,0 +1,162 @@
+apiVersion: v1alpha1
+kind: AgentConfig
+metadata:
+  name: myhub # need to match the same name put in install-config
+rendezvousIP: 192.168.56.25 # one of the master IP 
+hosts:
+
+  - hostname: master-0
```

Comment:
sed -i 's/master/control/g'

Reply:
should we used control instead of master ?

Reply:
done.